models/comments.model.js

```diff
@@ -0,0 +1,41 @@
+const db = require("../db/connection");
+const { fetchArticleById } = require("./articles.model");
+
+exports.fetchCommentsByArticleId = async (article_id) => {
+  try {
+    const articleCheck = await fetchArticleById(article_id);
+
```

Comment:
This is a great way to check if it's an article ID that exists in the database, however, in the spirit of MVC I would advise calling this function in the controller instead of this model

Reply:
Updated to call the function in the controller to follow the MVC design.